README.md

```diff
@@ -8,60 +8,122 @@
+| 6    | 3               | 7                                     | Modern browsers (IE 11+ in compatibility mode) |
+| 5    | 2               | 6                                     | Modern browsers                                |
+| 4    | 2               | 6                                     | IE 11+, RN w/o Proxy support                   |
 
-Class based components **are not supported** except using `<Observer>` directly in class `render` method. If you want to transition existing projects from classes to hooks, use [mobx-react 6+](https://github.com/mobxjs/mobx-react).
-
-See more at [the libraries overview](https://mobx-react.js.org/libraries).
+`mobx-react-lite` requires React 16.8 or higher.
 
 ## User Guide 👉 https://mobx.js.org/react/react-integration.html
 
-The site contains various examples and recipes for using MobX in React world. Feel free to contribute. The API reference of this package follows 👇.
+---
 
 ## API reference ⚒
 
-### **`observer<P>(baseComponent: FunctionComponent<P>, options?: IObserverOptions): FunctionComponent<P>`**
-
-```ts
-interface IObserverOptions {
-    // Pass true to wrap the inner component with React.forwardRef.
-    // It's false by the default.
-    forwardRef?: boolean
-}
-```
+### **`observer<P>(baseComponent: FunctionComponent<P>): FunctionComponent<P>`**
 
 The observer converts a component into a reactive component, which tracks which observables are used automatically and re-renders the component when one of these values changes.
+Can only be used for function components. For class component support see the `mobx-react` package.
 
 ### **`<Observer>{renderFn}</Observer>`**
 
-Is a React component, which applies observer to an anonymous region in your component.
+Is a React component, which applies observer to an anonymous region in your component. `<Observer>` can be used both inside class and function components.
+
+### **`useLocalObservable<T>(initializer: () => T, annotations?: AnnotationsMap<T>): T`**
+
+Creates an observable object with the given properties, methods and computed values.
+
+Note that computed values cannot directly depend on non-observable values, but only on observable values, so it might be needed to sync properties into the observable using `useEffect` (see the example below at `useAsObservableSource`).
+
+`useLocalObservable` is a short-hand for:
+
+`const state = useState(() => observable(initializer(), annotations, { autoBind: true }))[0]`
+
+### **`useStaticRendering(enable: true)`**
 
-### **`useObserver<T>(fn: () => T, baseComponentName = "observed", options?: IUseObserverOptions): T`**
+Call `useStaticRendering(true)` when running in an SSR environment, in which `observer` wrapped components should never re-render, but cleanup after the first rendering automatically. Use `isUsingStaticRendering()` to inspect the current setting.
+
+---
+
+## Deprecated APIs
```

Comment:
Personally, I wouldn't keep this whole section here at all. Or do you actually want to keep these around in V3 and remove it in a future V4?

Reply:
Yeah wanted to keep them around in v3, to not make migration too hard, and remove them in v4 only. Also nice to have the docs somewhere available in case people are looking at old code. So I think it is probably not too much in the way here.